tsconfig.json

```diff
@@ -11,7 +11,7 @@
     "sourceMap": true,
     "composite": true,
     "isolatedModules": true,
-
+    "typeRoots": ["./node_modules/@types", "./packages/*/node_modules/@types"],
```

Comment:
Lmk if this helps anyone's TS IDE intellisense

Reply:
Haven't noticed issues and this is pretty much doing the default as far as I can tell. Would probably rather remove at this time